### DISCUSSION TITLE
WSA on Surface Laptop 7

### DISCUSSION THREAD

**Discussion by NetworkGuy149** [MustardChef/WSABuilds#394] [Q&A] [answered]
I just got a new Surface Laptop 7 and found out that WSA does not work because the Surface is ARM based.

Will WSABuilds work on ARM machines?

**Comment by MustardChef** [comment, marked as answer]
Known issue. WSA is not working on any Snapdragon X Elite/Copilot+ PCs.
Both Official and Modded versions do not run.

I do not have a Snapdragon X Elite/Copilot+ PC, so I cannot even begin diagnose the issue.

### TLDR: No, WSA will not run on Copilot+ PCs and I cannot do anything to fix this, currently.

**Reply by NetworkGuy149** [reply]
Not the answer I was hoping for, but is it on the list to be addressed sometime in the future?

**Reply by MustardChef** [reply]
> Not the answer I was hoping for, but is it on the list to be addressed sometime in the future?

Maybe. I sure hope so. Microsoft has clearly changed something that has caused WSA break.
